src/components/Header/Header.module.scss

```diff
@@ -18,11 +15,11 @@
 
   .tab {
     transition: min-width 0.2s;
-    color: var(--mui-palette-primary-contrastText);
+    color: $neutral-white;
```

Comment:
The changes to the header got reverted somehow, please update to avoid reverting the updates.